polytiles/polytiles.py

```diff
@@ -124,9 +122,7 @@ def write_poly(self, poly):
         pass
 
     def tile_uri(self, x, y, z):
-        return '{0}{1}/{2}/{3}.{4}'.format(self.tile_dir,
-                                           z, x, y if not self.tms else 2**z-1-y,
-                                           format2ext(self.format))
+        return "{0}{1}/{2}/{3}.{4}".format(self.tile_dir, z, x, y if not self.tms else 2**z - 1 - y, format2ext(self.format))
```

Comment:
I guess this line won't pass the length limit for PEP8. I strive to limit lines to 100 characters max.